### PR TITLE
LPS-144391 search-experiences-web: Return full element JSON when clicking on  'View Element JSON'

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-144391 - "View Element JSON" should return the full JSON of the Element (with replaced variables)

Notes:
- Created the util `getSXPElementJSON` inside `util.js` so as not to clutter the SXPElement component
- Ended up alphabetizing some of the util functions related to elements and configurations

Thanks for taking a look!

### CODE REVIEW

File: modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/utils.js [hunk 1]
```diff
@@ -565,6 +565,34 @@ export function parseCustomSXPElement(sxpElement, uiConfigurationValues) {
 	}
 }
 
+/**
+ * Function that provides the element JSON, with title, description, and elementDefinition.
+ * The elementDefinition's configuration is updated to have its variables replaced with
+ * values from uiConfigurationValues.
+ *
+ * @param {object} _.sxpElement SXP Element with title, description, elementDefinition
+ * @param {object} _.uiConfigurationValues Values that will replace the keys in uiConfiguration
```
Comment (thektan) [inline modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/utils/utils.js:554]:
```suggestion
 * @param {object} sxpElement SXP Element with title, description, elementDefinition
 * @param {object} uiConfigurationValues Values that will replace the keys in uiConfiguration
```

**Review by thektan — CHANGES_REQUESTED** [review]
Just that one small doc correction.

Looks good though! I'm definitely in favor of this specific function with clear documentation of its purpose. I also appreciate you putting the formatting changes in a separate commit. 💯

**Comment by thektan** [comment]
@oliv-yu You can run ci:forward once you make that doc change. Thanks!

**Comment by oliv-yu** [comment]
ci:forward

**Pull request closed without merging**